Review section for Chart Control. (#609)
telerik/maui-docs: controls/chart/overview.md

# .NET MAUI Chart Overview

> Telerik UI for .NET MAUI Charts are feature-rich, intuitive, and easy to use data-visualization controls. While the Chart for .NET MAUI capitalizes all the innate benefits of the native UI, it exposes its objects and properties in C#, allowing for no-compromise customization and flexibility. Using Telerik Chart along with the .NET MAUI allows developers to implement various chart scenarios in their applications from a single shared C# code base.

Reviewer: ```suggestion
Telerik UI for .NET MAUI Charts are feature-rich, intuitive, and easy-to-use data-visualization controls. While the Chart for .NET MAUI capitalizes on all the innate benefits of the native UI, it exposes its objects and properties in C#, allowing for no-compromise customization and flexibility. Using the Telerik Chart along with .NET MAUI allows developers to implement various chart scenarios in their applications from a single shared C# code base.
```

### Behaviors

> Each Telerik UI for .NET MAUI Chart can be enabled with interactivity through its `Behaviors` property. A behavior an abstraction that handles user input in a `RadChart` instance and, optionally, provides visual feedback upon some action.

Reviewer: missing verb

The Telerik UI for .NET MAUI Charts support the following behaviors:

- [Pan-and-Zoom behavior]({% slug chart-behaviors-pan-and-zoom %})&mdash;This behavior handles `Manipulation` events and/or `MouseMove` and `MouseWheel` to enable the panning and zooming of the associated chart plot area.

> - [Tooltip behavior]({% slug chart-behaviors-tooltip %})&mdash;This behavior handles `Hold` and/or `MouseMove` events to enable context-sensitive information about a data point. It differs from the `TrackballBehavior` for visual information and trigger action.

Reviewer: ```suggestion
- [Tooltip behavior]({% slug chart-behaviors-tooltip %})&mdash;This behavior handles the `Hold` and/or `MouseMove` events to enable context-sensitive information about a data point. The Tooltip behavior differs from the `TrackballBehavior` in its visual information and trigger action.
```
